### PR TITLE
prepare for next 6.5.0

### PR DESCRIPTION
**Pull request by spirosoik**
#### Ticket Link
https://mattermost.atlassian.net/browse/DOPS-786

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by cpanato — APPROVED** [review]

**Review by amyblais — APPROVED** [review]

**Comment by cpanato** [comment]
@isacikgoz let us know if this is needed

**Comment by isacikgoz** [comment]
We don't need to update database version anymore, so this can be omitted. Let's close the PR after we update the workflow documentation.

**Comment by agnivade** [comment]
Yes! This was one of the benefits of moving to the new model. DB migrations aren't tied to the model version any more. We only just need to update the model version from now on.

@isacikgoz - I think we discussed this previously, but now we need to return the morph schema version in the "About Mattermost" modal. Did we file a ticket for that?

**Comment by hanzei** [comment]
Plugins rely on the server version getting updates as they check with `min_server_version` if the want to check if the server supports the plugin.

**Comment by agnivade** [comment]
@hanzei - Just to clarify, we are talking about the db schema version, not the server version. That remains unchanged.

**Comment by hanzei** [comment]
That means there are still plans to merge this PR as it is?

**Comment by hanzei** [comment]
Okay, thanks for the clarification @agnivade :+1:

**Comment by spirosoik** [comment]
@agnivade @isacikgoz could you have a final check here please?

File: store/sqlstore/upgrade.go [hunk 1]
```diff
@@ -15,6 +15,7 @@ import (
 
 const (
 	CurrentSchemaVersion   = Version640
+	Version650             = "6.5.0"
```
Comment (agnivade) [inline store/sqlstore/upgrade.go:18]: I think we can remove this as well.

**Review by isacikgoz — APPROVED** [review]
Good to go after removing the constant in the `upgrade.go` file.

**Comment by spirosoik** [comment]
@agnivade @isacikgoz last check please

**Review by hanzei — APPROVED** [review]

**Comment by hanzei** [comment]
I'm going to merge this PR as it's a straight forward change and there are four approvals. Hope you are fine that what @agnivade @isacikgoz.

**Pull request merged**